Gemfile

```diff
@@ -26,7 +26,12 @@ gem 'mongoid_userstamp', git: 'https://github.com/DCAFEngineering/mongoid_userst
                          branch: 'master' # adds created_by and updated_by timestamps
 gem 'mongo_session_store', '>= 3.1.0' # stores sessions in database for security
 gem 'enumerize' # Mongoid doesn't have enum out of the box, so we get it here
-gem 'mongoid_rails_migrations' # Mongoid also does not have migrations out of the box, so we get that here
+# gem 'mongoid_rails_migrations' # Mongoid also does not have migrations out of the box, so we get that here
```

Comment:
If we don't remove this, pg tries to run migrations twice. This should be fine since we never actually wrote a mongo migration.